features/5-basic-sso/api.md

```diff
@@ -0,0 +1,254 @@
+# API Design Overview
+
+Have official plugins for common SSO support; And also make it easy to enable and config them from Skygear Portal.
+
+Portal needs to have an interface for on/off and configuration of ID / Secret.
+
+## Scenario
```

Comment:
Would expect the Scenario to be expanded. For example, we have included 3rd Party SDK as an expanded use case during the discussion.